src/main/java/org/pac4j/demo/dw/Pac4JDemoApplication.java

```diff
@@ -54,5 +55,7 @@ protected Result check() throws Exception {
                 return Result.healthy();
             }
         });
+
+        new DbServer();
```

Comment:
oh yes, you are right ^^ today is not my day :)

ok for the close, too bad, but not very critical anyway